gobblin-cluster/src/main/java/org/apache/gobblin/cluster/GobblinHelixJobScheduler.java

```diff
@@ -333,6 +345,20 @@ public void handleNewJobConfigArrival(NewJobConfigArrivalEvent newJobArrival) {
   @Subscribe
   public void handleUpdateJobConfigArrival(UpdateJobConfigArrivalEvent updateJobArrival) {
     LOGGER.info("Received update for job configuration of job " + updateJobArrival.getJobName());
+    String jobName = updateJobArrival.getJobName();
+    boolean throttleEnabled = PropertiesUtils.getPropAsBoolean(updateJobArrival.getJobConfig(),
```

Comment:
usually booleans are easily identified with `is....` like `isThrottleEnabled`

Reply:
does this default to false if config is not provided? if not provide a default value

Reply:
Default is provided by GobblinClusterConfigurationKeys.DEFAULT_HELIX_JOB_SCHEDULING_THROTTLE_ENABLED_KEY in the config file, which is set to false. Should I add an additional default proof in this part?

Reply:
but you are not using that value here right? You want to provide `DEFAULT_HELIX_JOB_SCHEDULING_THROTTLE_ENABLED_KEY` here so value is used

Reply:
In the function of getPropAsBoolean, that is:
```
public static boolean getPropAsBoolean(   @NotNull  Properties properties,
  String key,
  String defaultValue )
```
so it will call `String.valueOf(GobblinClusterConfigurationKeys.DEFAULT_HELIX_JOB_SCHEDULING_THROTTLE_ENABLED_KEY) `as the default value if `GobblinClusterConfigurationKeys.HELIX_JOB_SCHEDULING_THROTTLE_ENABLED_KEY` is not assigned